cli/network.go

```diff
@@ -336,6 +336,10 @@ func hostNetworkingRequested(configNetNs string) (bool, error) {
 }
 
 func setupNetworkNamespace(config *vc.NetworkConfig) error {
+	if config.DisableNewNetNs {
+		return nil
```

Comment:
It'd be worth adding some log traces here to confirm that we're running without any netns!

Reply:
It might be overkill to add this to every log call, but due to the delicate nature of namespaces + golang, how about we log the namespace details at various points in the code [*]? We already log the pid so this will give more detail than a simple "changed namspace" log call:

```go
const nsPath = "/proc/self/ns"

func getNamespaces() (string, error) {
	files, err := ioutil.ReadDir(nsPath)
	if err != nil {
		return "", err
	}

	var ns []string

	for _, file := range files {
		filePath := path.Join(nsPath, file.Name())

		linkName, err := os.Readlink(filePath)
		if err != nil {
			return "", err
		}

		ns = append(ns, linkName)
	}

	sort.Sort(sort.StringSlice(ns))

	namespaces := strings.Join(ns, ",")

	return namespaces, nil
}
```

This will return something like:

```
cgroup:[4026531835],ipc:[4026531839],mnt:[4026531840],net:[4026532008],pid:[4026531836],pid:[4026531836],user:[4026531837],uts:[4026531838]
```

---
[*] - At runtime startup, whenever we change namespace, and in this function to show if we did or didn't switch ns.

Reply:
Yes, it's good to add some logs. But in this function we cannot get container id or sandbox id for log  fields.